src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -891,6 +967,11 @@ namespace AppInstaller::Repository
         auto source = sourceList.GetCurrentSource(name);
         THROW_HR_IF(APPINSTALLER_CLI_ERROR_SOURCE_NAME_ALREADY_EXISTS, source != nullptr);
 
+        // Check for a non-user tombstone; hidden source data that we don't want to collide.

```

Comment:
>non-user tombstone [](http://example.com/codeflow?start=23&length=18)

I don't remember we have code that supports user tombstone source, would just calling sourceList.GetSource be enough?